src/fidesops/task/graph_task.py

```diff
@@ -497,14 +531,16 @@ def run_access_request(
     traversal: Traversal = Traversal(graph, identity)
     with TaskResources(privacy_request, policy, connection_configs) as resources:
 
-        def start_function(seed: Dict[str, Any]) -> Callable[[], List[Dict[str, Any]]]:
+        def start_function(
+            seed: List[Dict[str, Any]]
+        ) -> Callable[[], List[Dict[str, Any]]]:
             """Return a function that returns the seed value to kick off the dask function chain.
 
             The first traversal_node in the dask function chain is just a function that when called returns
             the graph seed value."""
 
             def g() -> List[Dict[str, Any]]:
-                return [seed]
+                return seed
```

Comment:
This modification just allows us to just reuse this function for both root nodes and for the nodes that just return data they already have on restart.